lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -231,9 +254,13 @@ class MockTimers {
             this,
           );
         },
+        'Date.now': () => {
```

Comment:
Another possible option is to call the timer enum as `DateNow` to avoid using quoted properties in the object